src/pages/es/guides/integrations-guide.md

```diff
@@ -0,0 +1,155 @@
+# Usando PNPM
+pnpx astro add react
+```
+
+¡Incluso es posible configurar múltiples integraciones al mismo tiempo!
+
+```shell
+# Usando NPM
+npx astro add react tailwind partytown
+# Usando Yarn
+yarn astro add react tailwind partytown
+# Usando PNPM
+pnpx astro add react tailwind partytown
+```
+
+## Manejando las dependencias de integraciones
+
+Al instalar una integración de Astro en su proyecto, esté atento a las advertencias de "missing peer dependencies" que ve durante el proceso de instalación. No todos los gestores de paquetes instalarán automáticamente las peer-dependencias. Si está en Node v16+ y usa npm, no debería preocuparse por esta sección.
+
+```diff
+# Ejemplo: Instale las integraciones y frameworks juntos
+- npm install @astrojs/react
++ npm install @astrojs/react react react-dom
+```
+
+Si omite este paso, no se preocupe, Astro le advertirá al iniciar su entorno de desarrollo si se requieren dependencias que no se encuentran en su proyecto.
+
+Administrar sus propias peer-dependencias puede ser un poco más laborioso, pero también le permite controlar exactamente qué versiones de paquetes usa para cosas como React, Tailwind y más. Esto le da más control sobre su proyecto.
+
+En el futuro, el comando `astro add` podrá manejar toda esta configuración por usted e instalar automáticamente las peer-dependencias correctas para sus integraciones.
+
+## Usando integraciones
+
+Las integraciones de Astro siempre se agregan a través de la propiedad `integrations` en su archivo `astro.config.mjs`.
+
+:::tip[¿Necesita más información sobre una integración específica?]
+Encuéntrelo en nuestro [directorio de integraciones](https://astro.build/integrations/) y siga el enlace a su repositorio en GitHub para obtener instrucciones más detalladas de uso y configuración.
+:::
+
+Hay tres formas comunes de importar una integración a su proyecto Astro:
+1. Instalando el paquete npm de la integración.
+2. Importando su propia integración desde un archivo local dentro de su proyecto.
+3. Escribiendo su integración en línea, directamente en su archivo de configuración.
+
+```js
+// astro.config.mjs
+import {defineConfig} from 'astro/config';
+import installedIntegration from '@astrojs/vue';
+import localIntegration from './my-integration.js';
+
+export default defineConfig({
+  integrations: [
+    // 1. Imported from an installed npm package
+    installedIntegration(),
+    // 2. Imported from a local JS file
+    localIntegration(),
+    // 3. An inline object
+    {name: 'namespace:id', hooks: { /* ... */ }},
+  ]
+})
+```
+
+Consulte la referencia de [API de integración](/es/reference/integrations-reference/) para conocer las diferentes formas para escribir una integración.
+
+### Opciones personalizadas
+
+Las integraciones casi siempre se crean como funciones de fábrica que devuelven el objeto de integración real. Esto le permite pasar argumentos y opciones a la función de fábrica que personaliza la integración para su proyecto.
```

Comment:
I don't find "función de fábrica" like natural language but I cannot think of another thing to use here 🤔 maybe we could leave "factory function"? Or maybe "función factory"? What do you think?

Reply:
creo que mantenerla en ingles seria lo mejor